src/components/SocialMediaList/SocialMediaList.module.scss

```diff
@@ -0,0 +1,82 @@
+@use '../../styles/global';
+@use '../../styles/breakpoints.scss';
```

Comment:
Ainda to confunso, path alias ou nao? Confirma com o Ale se o path alias tá 100%, se estiver mudar pro path alias

Reply:
o do sass funciona certinho, precisa colocar o pathalias